website/index.html

```diff
@@ -4,118 +4,130 @@
+            </div>               
+            <div class="offer_item">
+                <svg class="offer_item_book"><use xlink:href="#Book"></use></svg>
+                <p class="offer_item_text">Book online</p>  
+            </div>    
+            <div class="offer_item">
+                <svg class="offer_item_real_guest"><use xlink:href="#Real"></use></svg>
+                <p class="offer_item_text">Real guest reviews</p>
             </div>
+            
         </div>
-    </div>
+    </section>
         <!--       Footer      -->
-        
-        <div class="footer">
-            <ul class="footer_column">
-                <li class="footer_title">About</li>
-                <li class="footer_text">How Triphouse works</li>
-                <li class="footer_text">Careers</li>
-                <li class="footer_text">Privacy</li>
-                <li class="footer_text">Terms</li>
-            </ul>
-            <ul class="footer_column">
-                <li class="footer_title">Property types</li>
-                <li class="footer_text">Guest houses</li>
-                <li class="footer_text">Hotels</li>
-                <li class="footer_text">Apartments</li>
-                <li class="footer_text">Villas</li>
-                <li class="footer_text">Holiday homes</li>
-                <li class="footer_text">Hostels</li>    
-            </ul>
-            <ul class="footer_column">
-                <li class="footer_title">Support</li>
-                <li class="footer_text">Contact Customer Service</li>
-                <li class="footer_text">FAQ</li>
-            </ul>
-            <div class="copyright_text">&#169; 2020 Triphouse, Inc. All rights reserved</div>
+    <footer class="footer">
+        <svg class="background_logo"><use xlink:href="#background_logo_vector"></use></svg>
+        <div class="footer_column">
+            <div class="footer_content">
+                <a href="#"><p>About</p></a>
+                <a href="#" class="footer_text">How Triphouse works</a>
+                <a href="#" class="footer_text">Careers</a>
+                <a href="#" class="footer_text">Privacy</a>
+                <a href="#" class="footer_text">Terms</a>
+            </div>
+            <div class="footer_content">
+                <a href="#"><p>Property types</p></a>
+                <a href="#" class="footer_text">Guest houses</a>
+                <a href="#" class="footer_text">Hotels</a>
+                <a href="#" class="footer_text">Apartments</a>
+                <a href="#" class="footer_text">Villas</a>
+                <a href="#" class="footer_text">Holiday homes</a>
+                <a href="#" class="footer_text">Hostels</a>
+            </div>
+            <div class="footer_content">
+                <a href="#"><p>Support</p></a>
+                <a href="#" class="footer_text">Contact Customer Service</a>
+                <a href="#" class="footer_text">FAQ</a>
+            </div>
```

Comment:
So large paddings for medium and small displays
![image](https://user-images.githubusercontent.com/20189313/129493072-45b9fa6e-a021-4fc0-aa25-8901b831cd93.png)